docs/pages/application-access/guides/aws-console.mdx

```diff
@@ -257,7 +258,7 @@ username which you can search for to get the events history:
 
 ## Step 8. Using AWS CLI
 
-Prequisites:
+Prequisite:
```

Comment:
```suggestion
Before beginning this step, make sure that the `aws` command line interface (CLI) tool is installed in PATH. For more information, read [Installing or updating the latest version of the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html).
```
If we're adding the prerequisite here, I suggest making it more conversational to fit with the tone of the body text.